aws/inputs.tf

```diff
@@ -81,8 +81,13 @@ variable "vpc_subnet_ids" {
   description = "A list of subnet IDs where the Auto Scaling Group can place the bastion."
 }
 
+variable "ssh_key_name" {
+  description = "Key name of an existing SSH aws_key_pair resource to use."
+  default     = ""
+}
+
 variable "ssh_public_key_file" {
-  description = "The content of an existing SSH public key file, that will be used to create an AWS SSH Key Pair. Yes, this input has an unfortunate name."
+  description = "The content of an existing SSH public key file, that will be used to create an AWS SSH Key Pair. If you generate it from terraform `aws_key_pair` resource, use `ssh_key_name` input instead. Yes, this input has an unfortunate name."
```

Comment:
This is only a suggestion, attempting to clarify the readability of this input description. I think the previous reference to "This input has an unfortunate name" is a bit confusing along with mentioning another input. Feel free to make your own adjustments.

```
The *content* of an existing SSH public key file, that will be used to create an AWS SSH Key Pair. This input is mutually exclusive with the `ssh_key_name` input, which can be used instead to reference a pre-existing Key Pair.
```

Reply:
I have updated the pull request with this sentence, thanks.